scipy/signal/splinemodule.c

```diff
@@ -45,13 +45,33 @@ convert_strides(npy_intp* instrides,npy_intp* convstrides,int size,int N)
 
 static char doc_cspline2d[] = "cspline2d(input {, lambda, precision}) -> ck\n"
 "\n"
-"  Description:\n"
+"    Coefficients for 2-D cubic (3rd order) B-spline.\n"
 "\n"
-"    Return the third-order B-spline coefficients over a regularly spacedi\n" 
-"    input grid for the two-dimensional input image.  The lambda argument\n" 
-"    specifies the amount of smoothing.  The precision argument allows specifying\n"
-"    the precision used when computing the infinite sum needed to apply mirror-\n"
-"    symmetric boundary conditions.\n";
+"    Return the third-order B-spline coefficients over a regularly spaced\n"
+"    input grid for the two-dimensional input image.\n"
+"\n"
+"    Parameters\n"
+"    ----------\n"
+"\n"
```

Comment:
`\n` between parameters and returns aren't necessary

Reply:
I removed these.